command/agent_test.go

```diff
@@ -489,6 +489,124 @@ listener "tcp" {
 	}
 }
 
+// TestAgent_NoAutoAuthTokenIfNotConfigured tests that API proxy will not use the auto-auth token
+// unless configured to.
+func TestAgent_NoAutoAuthTokenIfNotConfigured(t *testing.T) {
+	logger := logging.NewVaultLogger(hclog.Trace)
+	cluster := vault.NewTestCluster(t, nil, &vault.TestClusterOptions{
+		HandlerFunc: vaulthttp.Handler,
+	})
+	cluster.Start()
+	defer cluster.Cleanup()
```

Comment:
I discovered minimal test cluster recently and it's just another thing that saves a few lines.

```suggestion
	cluster := minimal.NewTestSoloCluster(t, nil)
```

If you don't want this, you can remove your `cluster.Start()` and it's a no-op.

Reply:
Thanks! Always forget about this but it makes tests nice and clean. Updating